scss/_patterns_heading-icon.scss

```diff
@@ -22,25 +22,26 @@
     height: map-get($icon-sizes, heading-icon--small);
     margin-bottom: 0;
     margin-right: $sph--large;
-    margin-top: map-get($nudges, h3-mobile);
+    margin-top: calc(map-get($nudges, h3-mobile) / 2); // just a quick fix, should be calculated based on line height and icon size
```

Comment:
I guess ideally instead of arbitrary pushing it, we should attempt to center the icon within line height of the heading, so the calculation would need to calculate the difference between icon size and line height and use half of that to position the icon. 

Maybe I'll just do it if that helps with the mobile positioning.

Reply:
No, the icon is not "within" a line height, it's an image in flex box, that has to be aligned relative to heading that follows it. And I don't think we should refactor the component now, we just need to adjust it to new heading sizes.

This component (heading icon) was causing problems before, both from design and development point of view, so we should probably have a look at deprecating or re-doing it with the component rebranding.